app/javascript/components/action/action-button.jsx

```diff
@@ -4,15 +4,28 @@ import style from './style.module.less';
 
 export const ActionButton = ({id, type, title, setActionItems}) => {
   const onClickHandler = async () => {
+    let actionItem;
```

Comment:
It's better to use plural variable `actionItems` because you keep list of items :)